README.md

```diff
@@ -8,6 +8,20 @@ This repository contains a certificate manager based on Certbot, using the Consu
 Running the unit tests
 ----------------------
 
+There are two options to run the unit tests:
+
+### Devstack way:
+
+To run the unit tests in a devstack (containers), you need to start the devstack:
```

Comment:
Can we replace "containers" with "docker" just to make it more obvious that you need to have docker installed for this to work?